velox/functions/CoverageUtil.h

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <string>
+#include <unordered_set>
+#include <vector>
+
+std::vector<std::string> readFunctionNamesFromFile(const std::string& filePath);
+
+class TablePrinter {
```

Comment:
Can this class be moved into .cpp file?

Reply:
Yes, you are right. The CoverageUtil.h only have three functions which are used outside.